lk2nd/device/dts/msm8909/msm8909-zte-sapphire.dts

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8909 0>;
+	qcom,board-id = <0x1000b 0x6e>, <0x1040b 0x6e>;
+
+	compatible = "qcom,msm8909-qrd", "qcom,msm8909", "qcom,qrd";
+};
+
+&lk2nd {
+	zte-sapphire {
+		model = "ZTE N818S (sapphire)";
+		compatible = "zte,sapphire";
+		lk2nd,match-panel;
+		lk2nd,dtb-files = "msm8909-zte-sapphire";
+
+		panel {
+			// FIXME: this panel does not exist in mainline
+			// mdss_dsi_hlt_jd9161_hsd_4p_fwvga_video
```

Comment:
No panel will exist in mainline until someone adds it. For lk2nd we generally add everything and it's implied not all of them will be present. Perhaps this comment meant something else?